README.md

```diff
@@ -5,7 +5,7 @@ This tool uses Google’s [Lighthouse](https://github.com/GoogleChrome/lighthous
 ## Installation
 
 ```bash
-npm install --save-dev lighthouse-keeper
+npm install --save-dev @sumcumo/lighthouse-keeper
```

Comment:
Should be `@sum.cumo/lighthouse-keeper` :) see e.g. https://www.npmjs.com/package/@sum.cumo/nuxt-styleguide